packages/react/ds/src/header/components/header-slot.tsx

```diff
@@ -58,6 +58,7 @@ const DrawerTrigger = ({
             })}
             icon={icon}
             id={`ItemIconDrawerActionTrigger-${index}`}
+            ariaHidden={!!label}
```

Comment:
```suggestion
            ariaHidden={true}
```
The value is the icon name and might differ from the label's meaning.